ligo/stablecoin/actions.ligo

```diff
@@ -240,4 +333,236 @@ function is_operator_action
   } with
   ( list [is_operator (parameter, store.operators)]
   , store
+  )
+
+(*
+ * Pauses whole contract in order to prevent all transferring, burning,
+ * minting and allowance operations. All other operations remain
+ * unaffected. Fails if the contract is already paused.
+ *)
+function pause
+  ( const parameter : pause_params
+  ; const store     : storage
+  ) : entrypoint is block
+{ ensure_not_paused (store)
+; authorize_pauser (store)
+} with
+  ( (nil : list (operation))
+  , store with record [paused = True]
+  )
+
+(*
+ * Unpauses whole contract so that all transferring, burning, minting and
+ * allowance operations can be performed. Fails if the contract is already
+ * unpaused.
+ *)
+function unpause
+  ( const parameter : unpause_params
+  ; const store     : storage
+  ) : entrypoint is block
+{ ensure_is_paused (store)
+; authorize_pauser (store)
+} with
+  ( (nil : list (operation))
+  , store with record [paused = False]
+  )
+
+(*
+ * Adds minter to minting allowances map setting its allowance
+ * to 0. Resets the allowance if associated address is already
+ * present. Fails if sender is not master minter.
+ * TODO
```

Comment:
TODO what?

Reply:
Implemented already, forgot to remove comment